Code/Framework/AzTest/AzTest/Platform/Linux/AzTest_Traits_Linux.h

```diff
@@ -13,8 +13,8 @@
 #define AZ_TRAIT_UNIT_TEST_ENTITY_ID_GEN_TEST_COUNT 10000
 #define AZ_TRAIT_UNIT_TEST_DILLER_TRIGGER_EVENT_COUNT 100000
 
-#define AZ_TRAIT_DISABLE_FAILED_AP_CONNECTION_TESTS true
-
+#define AZ_TRAIT_DISABLE_FAILED_ASSET_LOAD_TESTS true
+#define AZ_TRAIT_DISABLE_FAILED_ASSET_PROCESSOR_TESTS true
```

Comment:
Didn't you just remove this define in another PR?

Reply:
Looks like a bad merge resolve